src/shared/components/oncoprint/Oncoprint.tsx

```diff
@@ -108,16 +114,24 @@ interface IBaseHeatmapTrackSpec {
     data: IBaseHeatmapTrackDatum[];
     trackGroupIndex: number;
 }
-export interface IGeneHeatmapTrackSpec extends IBaseHeatmapTrackSpec {
-    data: IGeneHeatmapTrackDatum[];
-    onRemove: () => void;
+
+export interface IHeatmapTrackSpec extends IBaseHeatmapTrackSpec {
+    data: IBaseHeatmapTrackDatum[]; // can be IGeneHeatmapTrackDatum or ITreatmentHeatmapTrackDatum
     info?: string;
     labelColor?: string;
+    trackLinkUrl?: string | undefined;
+    onRemove: () => void;
+    molecularProfileName?: String
+    pivotThreshold?: number;
+    sortOrder?: SortOrder;
+    maxProfileValue?: number;
+    ruleSetTrackId?: number;
+    category?:string;
```

Comment:
what's the purpose of this prop?

Reply:
`sortOrder` controls the direction of the track legend. With SortOrder.ASC the smallest value is shown at the left side and gets the 'significant' color. With SortOrder.DESC the larges  value is shown at the left side and gets the 'significant' color.

`maxProfileValue` and `minProfileValue` (recently added in new commit) represent the max and min values all tracks of a trackGroup. This information is added because only a single track controls the legend, but the legend is dynamically drawn based on all tracks in the group.

Reply:
sorry, I should have been more specific: I meant the `category` prop

Reply:
Ah I see. Treatment profile data points in essence consist of a value (e.g. '8.00') and an optional  'thresholdType' (e.g. '>'). For Oncoprint trackGroups data points that have a thresholdType will have a `category` member that is a concatenation between the rounded value prefixed with the thresholdType (e.g., '>8.00'). This `category` member is used by oncoprintJS to draw the recently implemented 'gradient+categorical' legend type.

Reply:
I understand. But why is the category prop in the HeatmapTrackSpec?

Reply:
Was a remnant of earlier attempts. Field was removed.